rdkit/Chem/UnitTestRandomize.py

```diff
@@ -0,0 +1,48 @@
+import pytest
```

Comment:
Though we use pytest now to run tests, we may not do so forever and I want to future proof both code and tests as much as possible.

Please implement the tests using Python's standard unittest module.

Reply:
Makes sense, see [5458a3a](https://github.com/rdkit/rdkit/pull/7232/commits/5458a3ae8287adccd4d48a0bc621a65dd081c6b7).